deploy/helm/elastic-agent/templates/agent/eck/_pod_template.tpl

```diff
@@ -0,0 +1,82 @@
+{{- define "elasticagent.engine.eck.podTemplate" }}
```

Comment:
Does this need to be a template function? Could we not make a *.yaml file for this and include that in the manifests? It'd be nicer to work with due to syntax highlighting and such.

Reply:
look at the comment below